server/sonar-server/src/main/java/org/sonar/server/issue/IssueUpdater.java

```diff
@@ -257,7 +261,7 @@ public void setCloseDate(DefaultIssue issue, @Nullable Date d, IssueChangeContex
 
   public boolean setEffortToFix(DefaultIssue issue, @Nullable Double d, IssueChangeContext context) {
     if (!Objects.equal(d, issue.effortToFix())) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this use of "effortToFix"; it is deprecated. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)